minecraftServers/Servers/ServerList.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace minecraftServers.Servers;
```

Comment:
I think, it's better to change your folder structure. It's not really a folder for servers, it only contains server model, so how about `Models`?